main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"github.com/kataras/golog"
+	"github.com/vitwit/go-fass/openfass-go"
+)
+
+func main() {
+	cli := openfass_go.NewSendClient("admin", "9a651783248f8b61a92ae2fa02c7cb627a68e061f5cc0773822f72e0c7af077d",
```

Comment:
Use env variables instead

Reply:
****
Just for testing purpose i have given there. Particular user will give their credentials while creating client. I am thinking that no need to maintain .env as we are not storing any details and there won't be any main function also.